databrowser/src/domain/cellComponents/components/filters/CancelSaveButtons.vue

```diff
@@ -0,0 +1,21 @@
+<template>
```

Comment:
We normally name these buttons the `ControllButtons`. I don't know if you like the name

Reply:
Done with 295d9b6